src/output.h

```diff
@@ -208,4 +208,13 @@ void OutputLoggerExitPrintStats(ThreadVars *, void *);
 void OutputSetupActiveLoggers(void);
 void OutputClearActiveLoggers(void);
 
+typedef bool (*SimpleJsonTxLogFunc)(void *, struct JsonBuilder *);
```

Comment:
I don't like this "SimpleJson" name space. I would like to move all our Eve related logic to start using the `Eve` namespace. So I guess here we'd use `EveSimpleTxLogFunc`

Reply:
While I've moved some Json stuff to "Eve" in the past, I have recently been questioning this.  JSON logging is straightforward and requires no explanation. What is "Eve" logging? What does it stand for? :) It always requires more explanation than JSON logging.

Reply:
Renaming to `EveJsonSimpleTxLogFunc` then